gtsam/navigation/tests/testManifoldPreintegration.cpp

```diff
@@ -98,7 +98,9 @@ TEST(ManifoldPreintegration, computeError) {
   pim.computeError(x1, x2, bias, aH1, aH2, aH3);
   boost::function<Vector9(const NavState&, const NavState&,
                           const imuBias::ConstantBias&)> f =
-      boost::bind(&ManifoldPreintegration::computeError, pim, _1, _2, _3,
+      boost::bind(&ManifoldPreintegration::computeError, pim,
```

Comment:
It would be nice to have consistency amongst tests in the use of `using namespace boost::placeholders;` but whatever.

Reply:
I didn't because we don't use namespaces at all in this file.

Reply:
No reason not to add them for this case.